sklearn/base.py

```diff
@@ -250,6 +252,41 @@ def get_params(self, deep=True):
                 deep_items = value.get_params().items()
                 out.update((key + "__" + k, val) for k, val in deep_items)
             out[key] = value
+
```

Comment:
Let's not modify directly the `get_params` method such that it still return a normal Python dictionary. Instead, we could create a private method (e.g. _get_params` but maybe a more informative name), that we can call in `estimator_html_repr` instead of the actual `get_params()`.

Reply:
Done